ddtrace/sampler.py

```diff
@@ -340,5 +345,7 @@ def _choose_priority_category_with_rule(self, rule, sampler):
             return _PRIORITY_CATEGORY.RULE_DEF
 
         if self.limiter._has_been_configured:
+            # if the default rate limiter is used to sample traces
+            # auto_keep and auto_drop should be used
```

Comment:
```suggestion
            # If the default rate limiter is NOT used to sample traces
            # the sampling priority must be set to manual keep/drop.
            # This will disable agent based sample rates.
```